.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Is it possible to locally ignore this instead of adding it to the gitignore? That way we can add repo wide node modules if needed?

Reply:
Typically, you want to ignore this folder, what you as a developer would do is run `npm install` or the yarn equivalent before beginning work after pulling the source, when you add new modules, its added to package.json and then when pulled you'd run the same command again.

node_modules can get back and there are some platform dependent  things in there so you'd want to avoid having that folder distributed and instead you'd just distribute package.json and package-lock.json

Reply:
Nice ok then, disregard my comment in that case.